package.json

```diff
@@ -81,7 +81,8 @@
                         "bat": "",
                         "shellscript": "bash",
                         "fsharp": "fsi",
-                        "csharp": "scriptcs"
+                        "csharp": "scriptcs",
+                        "vbscript": "cscript //Nologo"
```

Comment:
Make sense! That needs to be taken into consideration. I guess you have installed [VBScript extension](https://marketplace.visualstudio.com/items?itemName=luggage66.VBScript) to get the VBScript setting. By default, VS Code only has Visual Basic language setting. And you need to change to VBScript every time open a vbs file even if you have installed the VBScript extension. That's not convenient. How about add a config entry like below to determine the executor per file extension?
```json
"code-runner.executorMapByFileExtension": {
        "vbs": "cscript //Nologo"
}
```